src/common/components/table-pk-picker/components/field-tree.business.tsx

```diff
@@ -1,12 +1,12 @@
 import { Tick } from '../../icons/tick-icon.component';
-import { OptionVm } from '../table-pk-picker.model';
+import { PkOptionVm } from '../table-pk-picker.model';
 import classes from '../table-pk-picker.component.module.css';
 
 export const generateOptions = (
```

Comment:
Add reference to this issue (unit tests):

https://github.com/Lemoncode/mongo-modeler/issues/116

Reply:
It's fixed